src/locale/properties/shared/shared.ca.properties

```diff
@@ -0,0 +1,338 @@
+shared.questionMark=?
+shared.ariaLabelClose=Close
+shared.hideDetails=Hide details
+shared.showDetails=Show details
+shared.edit=Edit
+shared.makeCopy=Make a copy and edit
+shared.saveChanges=Save changes
+shared.cancel=Cancel
+shared.visibility=Visibility
+shared.delete=Delete
+shared.showMore=Show more
+shared.of=of
+shared.funding=Funding
+shared.organizationIdentifiers=Organization identifiers
+shared.otherOrganization=Other organization identifiers provided by
+shared.translatedTitle=Translated title
+shared.added=Added
+shared.lastModified=Last modified
+shared.url=URL
+shared.employment=Employment
+shared.educationQualifications=Education and qualifications
+shared.qualification=Qualification
+shared.education=Education
+shared.invitedPositions=Invited positions and distinctions
+shared.invitedSentenceCase=Invited position
+shared.distinction=Distinction
+shared.membership=Membership and service
+shared.membershipTitle=Membership
+shared.service=Service
+shared.works=Works
+shared.lastModifiedTime=Record last modified
+shared.partOf=Part de
+shared.onBehalfOf=via
+shared.versionOf=Version of
+shared.add=Add
+shared.sort=Sort
+shared.addEmployment=Add Employment
+shared.addQualification=Add Qualification
+shared.addEducation=Add Education
+shared.addInvited=Add Invited Position
+shared.addDistinction=Add Distinction
+shared.addMembership=Add Membership
+shared.addService=Add Service
+shared.addFunding=Add Funding
+shared.addWork=Add Work
+shared.addPeerReviews=Add Peer Review
+share.loadMore=Load more
+share.sortTitle=Title
+share.sortStart=Start
+share.sortEnd=End
+share.sortDate=Date
+share.sortType=Type
+shared.addToList=Add to list
+shared.organization=Organization
+shared.organizationName=Organization name
+shared.department=Department
+shared.city=City
+shared.cityError=Si us plau entreu una ciutat
+shared.country=Country
+shared.role=Role/title
+shared.degree=Degree/title
+shared.award=Distinction/award
+shared.membershipType=Membership type
+shared.startDate=Start date
+shared.endDate=End date
+shared.year=Year
+shared.month=Month
+shared.day=Day
+shared.invalidDate=Data invàlida
+shared.actions=Actions
+shared.exportWorks=Export works
+shared.whoCanSeeThis=Who can see this?
+shared.present=present
+shared.to=to
+shared.searchLink=Search & link
+shared.addManually=Add manually
+shared.view=View
+shared.relationship=Relació
+shared.source=Source
+shared.emails=Emails
+shared.publicationDate=Publication date
+shared.selectACountry=Select a country
+shared.trustedParties=Trusted parties
+shared.onlyMe=Only me
+shared.everyone=Everyone
+shared.maxLength.less_than_one_thousand=Ha de tenir menys de 1000 caràcters
```

Comment:
@leomendoza123 Could you remove this unused properties

Reply:
This one are used, since I am cloning translations these ones are showing up now. 
We use them for the reset password.

Reply:
You are right! Removing...